src/screens/ProfileScreen/index.js

```diff
@@ -605,7 +605,7 @@ const ProfileScreen = ({route}) => {
   };
 
   return (
-    <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
+    <View style={styles.container} forceInset={{top: 'always'}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `SafeAreaProvider` component has been replaced with a `View` component. This could potentially affect the layout on devices with notches or rounded corners.
```diff
- <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
+ <View style={styles.container} forceInset={{top: 'always'}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->